engine/src/main/java/org/terasology/rendering/nui/widgets/UIText.java

```diff
@@ -667,7 +674,12 @@ public void bindText(Binding<String> binding) {
      * @return The text contained by the text box
      */
     public String getText() {
-        return text.get();
+        String arrayText = "";
+        if(textArr != null){
+            for(int i = 0;i < textArr.length;i++) arrayText += textArr[i] + "\n";
+        }
+        if(text.get() != "") return text.get() + "\n" + arrayText;
+        return text.get() + arrayText;
```

Comment:
I don't know if this behavior (of concatenating `text` and `arrayText` in that order) should be standard or not (I don't think it should because IMO _either_ `textArr` or `text` should be used at a given time), but if it is, it should be documented somewhere.

Reply:
Also, this will cause an NPE given that setting the text array sets `text` to null.